repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1351,3 +1351,18 @@ import.zip.compressionRatioThreshold=100
 # "zip bomb" and the import extraction process cancelled. No value (or a negative long) will be taken to mean that no
 # limit should be applied.
 import.zip.uncompressedBytesLimit=
+
+# Rhino optimization level
+scripts.execution.optimizationLevel=0
+
+# Max seconds a script is allowed to run
+scripts.execution.maxScriptExecutionSeconds=-1
+
+# Max call stack depth
+scripts.execution.maxStackDepth=-1
+
+# Max memory (bytes) a script can use
+scripts.execution.maxMemoryUsed=-1
```

Comment:
Same here, changing the name to `maxMemoryUsedInBytes` is more convenient and less error prone without reading the comment!

Reply:
Added `InBytes` suffix to every occurrence of `maxMemoryUsed` property.